docs/development/standards.rst

```diff
@@ -1,159 +1,75 @@
-runner that accumulates these files.
-
-Deployment
-----------
-
-If merging several branches with JavaScript changes, it's important to do a
-final post-merge bundle. Follow the steps above to rebundle the libraries, and
-check in any changed libraries.
-
-JavaScript Bundles
-------------------
-
-There are several components to our bundling scheme:
-
-    Vendor library
-        We repackage these using `Browserify`_, `Bower`_, and `Debowerify`_ to
-        make these libraries available by a ``require`` statement.  Vendor
-        libraries are packaged separately from our JavaScript libraries, because
-        we use the vendor libraries in multiple locations. Libraries bundled
-        this way with `Browserify`_ are available to our libraries via
-        ``require`` and will back down to finding the object on the global
-        ``window`` scope.
-
-        Vendor libraries should only include libraries we are commonly reusing.
-        This currently includes `jQuery` and `Knockout`. These modules will be
-        excluded from libraries by special includes in our ``gulpfile.js``.
-
-    Minor third party libraries
-        These libraries are maybe used in one or two locations. They are
-        installed via `Bower`_ and included in the output library file. Because
-        we aren't reusing them commonly, they don't require a separate bundle or
-        separate include. Examples here would include jQuery plugins used on one
-        off forms, such as jQuery Payments.
-
-    Our libraries
-        These libraries are bundled up excluding vendor libraries ignored by
-        rules in our ``gulpfile.js``. These files should be organized by
-        function and can be split up into multiple files per application.
-
-        Entry points to libraries must be defined in ``gulpfile.js`` for now. We
-        don't have a defined directory structure that would make it easy to
-        imply the entry point to an application library.
-
-.. _`Bower`: http://bower.io
-.. _`Gulp`: http://gulpjs.com
-.. _`Browserify`: http://browserify.org
-.. _`Debowerify`: https://github.com/eugeneware/debowerify
-.. _`LESS`: http://lesscss.org
 
-.. _`jQuery`: http://jquery.com
-.. _`Knockout`: http://knockoutjs.com
+These are build standards that are adhered to by the core development team while
+developing Read the Docs and related services. If you are a new contributor to
+Read the Docs, it might a be a good idea to follow these guidelines as well. The
+:doc:`standard installation instructions <../install>` do cover what it takes to
+get started with a local installation of Read the Docs and can be used for local
+development by non-core team.
+
+Core team standards
+-------------------
+
+Core team members expect to have a development environment that closely
+approximates our production environment, in order to spot bugs and logical
+inconsistencies before they make their way to production.
+
+Celery runs as a separate process
```

Comment:
maybe mention rdb here

Reply:
`rdb` is mentioned in the supervisord section. Maybe it's enough? or maybe it's better to move that part into an specific section?

Reply:
@stsewd has a good point about having a debug guide for celery separate processes in local development. If you're used to running with always eager, your debugging workflow is going to change pretty dramatically here.

On `rdb` + supervisord, I'd like to spend some more time on this at some point and figure out if there is a better way to run all these processes in a way that makes it easier to debug. Too bad there isn't some application that combines tmux + supervisord -- something like this maybe: https://gist.github.com/SmallRoomLabs/1164361